src/main/java/com/conveyal/r5/profile/FastRaptorWorker.java

```diff
@@ -316,9 +316,16 @@ private void advanceScheduledSearchToPreviousMinute (int nextMinuteDepartureTime
 
             return result;
         } else {
-            // no frequencies, return result of scheduled search
-            if (saveAllStates) statesEachIteration.add(scheduleState[request.maxRides].deepCopy());
-            return new int[][] { scheduleState[request.maxRides].bestNonTransferTimes };
+            // No frequencies, return result of scheduled search, but duplicated by the number of
+            // MC draws so that the scheduled search accessibility isn't artificially deflated
+            // relative to any networks with frequencies.
+            int[][] result = new int[iterationsPerMinute][];
+            for (int i = 0; i < monteCarloDrawsPerMinute; i++) {
+                if (saveAllStates) statesEachIteration.add(scheduleState[request.maxRides].deepCopy());
+                result[i] = scheduleState[request.maxRides].bestNonTransferTimes;
+            }
+            if (saveAllStates) statesEachIteration.add(scheduleState[request.maxRides * monteCarloDrawsPerMinute].deepCopy());
```

Comment:
I don't think we want to add it to statesEachIteration again outside the loop, this will result in statesEachIteration being longer than the travel time results.